### PR TITLE
pyats:add new page

### PR DESCRIPTION
**Pull request by barweiss45**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/pyats.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`pyats --help`
+
+- Show help on a pyATS subcommand (like `create`, `shell`, `diff`, `parse`, etc.):
+
+`pyats {{subcommand}} --help`
+
+- Execute a pyATS subcommand:
+
+`pyats {{subcommand}} [options]`
```
Comment (CleanMachine1) [inline pages/common/pyats.md:20]:
```suggestion
`pyats {{subcommand}}`
```

File: pages/common/pyats.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Show general help including available subcommands:
+
+`pyats --help`
+
+- Show help on a pyATS subcommand (like `create`, `shell`, `diff`, `parse`, etc.):
+
+`pyats {{subcommand}} --help`
+
+- Execute a pyATS subcommand:
```
Comment (CleanMachine1) [inline pages/common/pyats.md:18]:
```suggestion
- Run a `pyATS` subcommand:
```

File: pages/common/pyats.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Show the installed version of pyATS and libraries:
+
+`pyats version check`
+
+- Show general help including available subcommands:
+
+`pyats --help`
+
+- Show help on a pyATS subcommand (like `create`, `shell`, `diff`, `parse`, etc.):
```
Comment (CleanMachine1) [inline pages/common/pyats.md:14]:
```suggestion
- Display help about a specific subcommand:
```

File: pages/common/pyats.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# pyATS
+
+> A vendor agnostic test automation framework by Cisco Systems, used for network and systems testing.
+> More information: <https://developer.cisco.com/pyats/>.
+
+- Show the installed version of pyATS and libraries:
+
+`pyats version check`
+
+- Show general help including available subcommands:
```
Comment (CleanMachine1) [inline pages/common/pyats.md:10]:
```suggestion
- Display help:
```

Could you move this example to the bottom

File: pages/common/pyats.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# pyATS
+
+> A vendor agnostic test automation framework by Cisco Systems, used for network and systems testing.
+> More information: <https://developer.cisco.com/pyats/>.
+
+- Show the installed version of pyATS and libraries:
```
Comment (CleanMachine1) [inline pages/common/pyats.md:6]: As well as this one.
```suggestion
- Display version:
```

File: pages/common/pyats.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`pyats --help`
+
+- Display help about a specific subcommand:
+
+`pyats {{subcommand}} --help`
+
+- Run a `pyATS` subcommand:
+
+`pyats {{subcommand}}`
```
Comment (kbdharun) [inline pages/common/pyats.md:20]:
```suggestion
`pyats {{subcommand}}`

- Display version:

`pyats version check`

- Display help:

`pyats --help`
```

File: pages/common/pyats.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# pyATS
+
+> A vendor agnostic test automation framework by Cisco Systems, used for network and systems testing.
+> More information: <https://developer.cisco.com/pyats/>.
+
+- Display version:
```
Comment (kbdharun) [inline pages/common/pyats.md:6]:
```suggestion

```

File: pages/common/pyats.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
```
Comment (kbdharun) [inline pages/common/pyats.md:7]:
```suggestion
```

File: pages/common/pyats.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+`pyats version check`
```
Comment (kbdharun) [inline pages/common/pyats.md:8]:
```suggestion

```

File: pages/common/pyats.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
```
Comment (kbdharun) [inline pages/common/pyats.md:9]:
```suggestion
```

File: pages/common/pyats.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+- Display help:
```
Comment (kbdharun) [inline pages/common/pyats.md:10]:
```suggestion
```

File: pages/common/pyats.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+
```
Comment (kbdharun) [inline pages/common/pyats.md:11]:
```suggestion
```

File: pages/common/pyats.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+`pyats --help`
```
Comment (kbdharun) [inline pages/common/pyats.md:12]:
```suggestion
```

File: pages/common/pyats.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+
```
Comment (kbdharun) [inline pages/common/pyats.md:13]:
```suggestion
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr @barweiss45, Thanks for your contribution, I have a suggestion we can rearrange the version and help command to the end.

**Review by kbdharun — COMMENTED** [review]
Hey, now that you have moved the `help` and `version` commands to last we can remove them from above.

File: pages/common/pyats.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
+# pyATS
+
+> A vendor agnostic test automation framework by Cisco Systems, used for network and systems testing.
+> More information: <https://developer.cisco.com/pyats/>.
+
+
```
Comment (kbdharun) [inline pages/common/pyats.md:6]:
```suggestion
```

**Review by kbdharun — APPROVED** [review]

**Comment by barweiss45** [comment]
> Hey, now that you have moved the `help` and `version` commands to last we can remove them from above.

I have moved the subcommand help entry to the bottom as well. Let me know how that looks. Thanks! 😀

**Review by CleanMachine1 — APPROVED** [review]
Thanks for your contribution.

**Comment by barweiss45** [comment]
Thank you, @CleanMachine1  and @kbdharun, I am very happy to be able to contribute. TLDR has been one of my regularly used CLI tools for some time now. It has been extremely valuable to me and helps me daily with my work. 😀

**Pull request merged**